ci_scripts/requirements-stdb.txt

```diff
@@ -0,0 +1,3 @@
+-e .
```

Comment:
Wonder if using full name instead of `-stdb.txt` wouldn't be better for future proofing when we come back to this.

Reply:
That crossed my mind too - I'll change it to use the full name.